apps/system/build/build.js

```diff
@@ -146,6 +146,35 @@ SystemAppBuilder.prototype.enableFirefoxSync = function(options) {
   preprocessor.execute(options, 'FIREFOX_SYNC', fileList);
 };
 
+SystemAppBuilder.prototype.disableBluetoothFeatures = function(options) {
+  var fileList = {
+    process: [
+      ['index.html'],
+      ['js', 'quick_settings.js']
+    ],
+    remove: {
+      'ifdef': [
+        ['js', 'bluetooth.js'],
+        ['js', 'bluetooth_core.js'],
+        ['js', 'bluetooth_headphone.js'],
+        ['js', 'bluetooth_icon.js'],
+        ['js', 'bluetooth_transfer.js'],
+        ['js', 'bluetooth_transfer_icon.js'],
+        ['js', 'bluetooth_transfer_v2.js'],
+        ['test', 'unit', 'bluetooth_core_test.js'],
+        ['test', 'unit', 'bluetooth_core_v2_test.js'],
+        ['test', 'unit', 'bluetooth_headphone_icon_test.js'],
+        ['test', 'unit', 'bluetooth_icon_test.js'],
+        ['test', 'unit', 'bluetooth_test.js'],
+        ['test', 'unit', 'bluetooth_transfer_icon_test.js'],
+        ['test', 'unit', 'bluetooth_transfer_test.js'],
+        ['test', 'unit', 'bluetooth_v2_test.js']
+      ]
```

Comment:
for NFC handover function `nfc_handover_manager.js` which is totally rely on bluetooth transfer, should we disable NFC transfer as well?